src/scripts/createMarkdown.sh

```diff
@@ -5,42 +5,42 @@ set -e
 
 # Ensure two arguments are provided
 if [ "$#" -ne 2 ]; then
-   echo "Usage: $0 <filename> <relative-path>"
-   exit 1
+  echo "Usage: $0 <filename> <relative-path>"
+  exit 1
 fi
 
 # Define arguments
 FILENAME="$1.md"
 RELATIVE_PATH="$2"
-BASE_PATH="public/content/platforms"
+BASE_PATH="public/content/documentation"
 
 # Construct the full path
 FULL_PATH="${BASE_PATH}/${RELATIVE_PATH}"
 
 # Check if the directory exists, if not create it
 if [ ! -d "$FULL_PATH" ]; then
-   mkdir -p "$FULL_PATH"
-   echo "Created directory: $FULL_PATH"
+  mkdir -p "$FULL_PATH"
+  echo "Created directory: $FULL_PATH"
 fi
 
 # Create the markdown file if it doesn't already exist
 FILE_PATH="${FULL_PATH}/${FILENAME}"
 if [ -f "$FILE_PATH" ]; then
-   echo "Error: File '$FILE_PATH' already exists!"
-   exit 1
+  echo "Error: File '$FILE_PATH' already exists!"
+  exit 1
 fi
 
 # Create the markdown file with a template
 cat >"$FILE_PATH" <<"EOF"
-# General Notes
+## General Notes
 > This file was generated using the createMarkdown script.
```

Comment:
Can you remove the ">" here? It renders like:
![Uploading Screenshot 2025-04-08 at 3.25.36 PM.png…]()